webhook/psbinding/reconciler.go

```diff
@@ -43,6 +43,12 @@ import (
 	"knative.dev/pkg/tracker"
 )
 
+var jsonLabelPatch = map[string]interface{}{
+	"metadata": map[string]interface{}{
+		"labels": map[string]string{"bindings.knative.dev/include": "true"},
```

Comment:
Want to use https://github.com/knative/pkg/blob/master/apis/duck/const.go#L32  constant, instead of this string?